### DISCUSSION TITLE
How to set font size?

### DISCUSSION THREAD

**Discussion by xuesoso** [Milkdown/milkdown#932] [Q&A]
Hi, I noticed the markdown font size setting does not change font size in Milkdown. I am wondering what setting option changes the font size? Thanks!

**Comment by ylc395** [comment]
what is "markdown font size setting"? Can you tell us how you try to set font size now?

**Reply by xuesoso** [reply]
There is an option for "markdown preview font size". I don't see any other option for changing font size related to markdown.

<img width="1379" alt="image" src="https://user-images.githubusercontent.com/15006047/227697683-6d9713a3-5fe8-4eb1-a7f6-1f6d6ccfef59.png">
